src/Tribe/Tickets.php

```diff
@@ -2473,7 +2473,7 @@ public function maybe_redirect_to_attendees_registration_screen( $redirect = nul
 			$storage = new Tribe__Tickets_Plus__Meta__Storage();
 			if ( ! empty( $redirect ) ) {
 				$key = $storage->store_temporary_data( $redirect );
-				$url = add_query_arg( array( 'event_tickets_redirect_to' => $key ), $url );
+				$url = add_query_arg( array( 'event_tickets_redirect_to' => $key, 'provider' => 'tribe_tpp_attendees' ), $url );
```

Comment:
Dont we have a variable holding this `tribe_tpp_attendees` string?

Reply:
```suggestion
				$commerce_paypal = tribe( 'tickets.commerce.paypal' );

				$url = add_query_arg( array( 'event_tickets_redirect_to' => $key, 'provider' => $commerce_paypal->attendee_object ), $url );
```